src/Entity/TaskSetInterface.php

```diff
@@ -7,12 +7,60 @@
 
 namespace SprykerSdk\SdkContracts\Entity;
 
+/**
+ * Defines the task set that can be used as a single task.
+ */
 interface TaskSetInterface extends TaskInterface
 {
     /**
+     * Specification:
+     * - Gets the list of task set tasks.
+     *
+     * @api
+     *
      * @param array<string> $tags
      *
      * @return array<\SprykerSdk\SdkContracts\Entity\TaskInterface|string>
      */
     public function getSubTasks(array $tags = []): array;
+
+    /**
+     * Specification:
+     * - Overrides tags of sub-tasks command property.
```

Comment:
It's just a getter. why doesn't this method just return a value but do something else? The same  for the rest of the methods with `makes` or `overrides` words in the specification